ignite/cmd/plugin.go

```diff
@@ -7,9 +7,12 @@ import (
 	"strings"
 	"time"
 
+	"github.com/ignite/cli/ignite/config"
```

Comment:
Import line should be moved to the lower import group

Reply:
https://github.com/ignite/cli/pull/3184/commits/e6858996520488ad1297add95c1003835fca0cd4